app/presenters/date_question_presenter.rb

```diff
@@ -15,15 +13,24 @@ def response_label(value)
   end
 
   def selected_day
-    prefill_value_for(self, :day)
+    return if response_for_current_question.blank?
```

Comment:
Minor: you don't actually need these guard conditions now you've got the second one. Sorry I didn't twig on that when I first made the hash suggestion.

Reply:
Thanks, I've fixed and fixed-up.